build-system/tasks/compile.js

```diff
@@ -136,6 +136,7 @@ function compile(entryModuleFilename, outputDir,
       // Not sure what these files are, but they seem to duplicate code
       // one level below and confuse the compiler.
       '!node_modules/core-js/modules/library/**.js',
+      'node_modules/resourcetiming-compression/src/resourcetiming-compression.js',
```

Comment:
Also, @erwinmombay, what's our current 3rd-party code linking policy? Do we do it via npm or `third_party`?